locked-asset/lkmex-transfer/src/lib.rs

```diff
@@ -0,0 +1,88 @@
+#![no_std]
+#![allow(clippy::vec_init_then_push)]
+
+elrond_wasm::imports!();
+elrond_wasm::derive_imports!();
+
+pub mod constants;
+use crate::constants::*;
+
+#[elrond_wasm::contract]
+pub trait LkmexTransfer {
+    #[init]
+    fn init(&self, unlock_transfer_time: u64, address_cooldown_duration: u64) {
```

Comment:
Rename `address_cooldown_duration` to something like `epochs_cooldown_duration`, or similar, to make it obvious it's epochs, and not something else (like blocks, timestamp, etc).

Same in other places it's being used.

Reply:
To help with this, maybe also have an alias like `type Epoch = u64`, and use that instead of u64, to help with code readability.